hawkular-inventory-bus-api/src/main/java/org/hawkular/inventory/bus/api/InventoryEvent.java

```diff
@@ -154,6 +156,14 @@ public void setAction(Action.Enumerated action) {
         this.action = action;
     }
 
+    public Tenant getTenant() {
+        return tenant;
+    }
+
+    public void setTenant(Tenant tenant) {
```

Comment:
Me neither, I just wanted to be consistent with the rest of the class, because there is also `{get|set}Object()`.

I can remove the setters for tenant and object. They are not being used anyway.